tc_driver/Cases.jl

```diff
@@ -312,19 +290,18 @@ function initialize_profiles(
     @. p_c = prof_p(z)
 end
 
-function surface_params(case::Bomex, surf_ref_thermo_state, thermo_params)
-    p_f_surf = TD.air_pressure(thermo_params, surf_ref_thermo_state)
-    ρ_f_surf = TD.air_density(thermo_params, surf_ref_thermo_state)
-    FT = eltype(p_f_surf)
+function surface_params(case::Bomex, thermo_params)
+    FT = eltype(thermo_params)
     zrough::FT = 1.0e-4
+    psurface::FT = 1.015e5
     qsurface::FT = 22.45e-3 # kg/kg
-    θ_surface::FT = 299.1
+    θ_surface::FT = 299.1 # PAPER CHECK: Should we use Tsurface = 300.4 instead?
     θ_flux::FT = 8.0e-3
     qt_flux::FT = 5.2e-5
-    ts = TD.PhaseEquil_pθq(thermo_params, p_f_surf, θ_surface, qsurface)
-    Tsurface = TD.air_temperature(thermo_params, ts)
-    lhf = qt_flux * ρ_f_surf * TD.latent_heat_vapor(thermo_params, ts)
-    shf = θ_flux * TD.cp_m(thermo_params, ts) * ρ_f_surf
+    ts = TD.PhaseEquil_pθq(thermo_params, psurface, θ_surface, qsurface)
```

Comment:
Just as a sanity check, could we once print here that TD.air_pressure(ts) and TD.air_temperature(ts) are as prescribed in the paper?